packages/map/src/map/utils.jsx

```diff
@@ -44,11 +43,14 @@ export const getMapOptions = ({
 
     scrollZoom: false,
     dragRotate: false,
+    dragPan: {
+      // Disable intertia on drag
```

Comment:
what's `intertia` ?

Reply:
by default mapbox adds a very laggy scroll inertia that does more harm than good